jekyll/_cci2/runner-installation.adoc

```diff
@@ -0,0 +1,465 @@
+```
+
+=== Platform-specific instructions
+
+Please refer to the platform-specific installation instructions:
+
+* linux
+* macOS
+
+== Installing the CircleCI runner on Linux
+
+=== Create the CircleCI runner configuration
+
+The recommended CircleCI runner configuration for Linux is:
+
+```yaml
+api:
+  auth_token: AUTH_TOKEN
+runner:
+  name: RUNNER_NAME
+  command_prefix: ["/opt/circleci/launch-task"]
+  working_directory: /opt/circleci/workdir/%s
+  cleanup_working_directory: true
+```
+
+=== Install the CircleCI runner configuration
+
+Once created, save the configuration file to `/opt/circleci/launch-agent-config.yaml` owned by `root` with permissions `600`.
+
+```bash
+sudo chown root: /opt/circleci/launch-agent-config.yaml
+sudo chmod 600 /opt/circleci/launch-agent-config.yaml
+```
+
+=== Create the circleci user & working directory
+
+These will be used when executing the `build-agent`.
+
+```bash
+id -u circleci &>/dev/null || adduser --uid 1500 --disabled-password --gecos GECOS circleci
+
+mkdir -p /opt/circleci/workdir
+chown -R circleci /opt/circleci/workdir
+```
+
+=== Install the launch script
+
+This wrapper script will be used by Launch Agent to execute the Task Agent, while ensuring appropriate sandboxing and a clean shutdown.
+
+Create `/opt/circleci/launch-task` owned by `root` with permissions `755`
+
+```bash
+#!/bin/bash
+
+set -euo pipefail
+
+## This script launches the build-agent using systemd-run in order to create a
+## cgroup which will capture all child processes so they're cleaned up correctly
+## on exit.
+
+# The user to run the build-agent as - must be numeric
+USER_ID=$(id -u circleci)
+
+# Give the transient systemd unit an inteligible name
+unit="circleci-$CIRCLECI_LAUNCH_ID"
+
+# When this process exits, tell the systemd unit to shut down
+abort() {
+  if systemctl is-active --quiet "$unit"; then
+    systemctl stop "$unit"
+  fi
+}
+trap abort EXIT
+
+systemd-run \
+    --pipe --collect --quiet --wait \
+    --uid "$USER_ID" --unit "$unit" -- "$@"
+```
+
+=== Enable the `systemd` unit
+
+Create `/opt/circleci/circleci.service` owned by `root` with permissions `755`.
+
+You must ensure that `TimeoutStopSec` is greater than the total amount of time the `stop-agent` script will take.
+
+If you want to configure the CircleCI runner installation to start on boot, it is important to note that the Launch Agent will attempt to consume and start jobs as soon as it starts, so it should be configured appropriately before starting. The Launch Agent may be configured as a service and be managed by systemd with the following scripts:
+
+```
+[Unit]
+Description=CircleCI Runner
+After=network.target
+[Service]
+ExecStart=/opt/circleci/circleci-launch-agent --config /opt/circleci/launch-agent-config.yaml
+Restart=always
+User=root
+NotifyAccess=exec
+TimeoutStopSec=600
```

Comment:
Could we change this to 18300 please?

for https://circleci.atlassian.net/browse/CIRCLE-31045